repo/packages/M/mariadb/0/package.json

```diff
@@ -5,7 +5,7 @@
   "scm": "https://github.com/MariaDB/server.git",
   "maintainer": "containers@bitnami.com",
   "website": "https://mariadb.org",
-  "description": "MariaDB is one of the most popular database servers in the world. It’s made by the original developers of MySQL and guaranteed to stay open source. Notable users include Wikipedia, Facebook and Google. MariaDB is developed as open source software and as a relational database it provides an SQL interface for accessing data. The latest versions of MariaDB also include GIS and JSON features.",
+  "description": "MariaDB is one of the most popular database servers in the world. It's made by the original developers of MySQL and guaranteed to stay open source. Notable users include Wikipedia, Facebook and Google. MariaDB is developed as open source software and as a relational database it provides an SQL interface for accessing data. The latest versions of MariaDB also include GIS and JSON features.",
```

Comment:
I can pull it out, but I'll have to disable my git commit hooks. When I run `./scripts/build.sh`, I get:

```
Validating mariadb...
	Validating revision 0...
		package.json:Traceback (most recent call last):
  File "/Users/sig/code/universe/scripts/validate-packages.py", line 123, in <module>
    sys.exit(main())
  File "/Users/sig/code/universe/scripts/validate-packages.py", line 36, in main
    _validate_package(given_package, package_path)
  File "/Users/sig/code/universe/scripts/validate-packages.py", line 44, in _validate_package
    _validate_revision(given_package, rev, os.path.join(path, rev))
  File "/Users/sig/code/universe/scripts/validate-packages.py", line 55, in _validate_revision
    package_json = _validate_json(package_json_path, PACKAGE_JSON_SCHEMA)
  File "/Users/sig/code/universe/scripts/validate-packages.py", line 109, in _validate_json
    data = json.loads(f.read())
  File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 290: ordinal not in range(128)
```